packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -158,13 +157,21 @@ class LinearProgressIndicator extends ProgressIndicator {
   /// The [value] argument can be either null (corresponding to an indeterminate
   /// progress indicator) or non-null (corresponding to a determinate progress
   /// indicator). See [value] for details.
+  ///
+  /// If [height] is less than 1.0 then a value of 1.0 will be used.
   const LinearProgressIndicator({
     Key key,
     double value,
     Color backgroundColor,
     Animation<Color> valueColor,
+    this.height,
```

Comment:
we should give this a default value of 6.0 in the constructor, then assert it's not null (and mention that it musn't be null in the constructor's documentation)

Reply:
Will do!